### PR TITLE
Release v2.15.0

### PR DESCRIPTION
**Pull request by LeoColomb**
🚀

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+  [[002a110](https://github.com/h5bp/server-configs-apache/commit/002a110bf35c25af66ab09ef1bd724ece5fd8266)].
+* Block Mercurial `.orig` files
+  [[4c13648](https://github.com/h5bp/server-configs-apache/commit/4c1364885477e836fec24a6d8330cba69cf3d3a0)].
+* Fix enforcing www/no-www with HTTPS
+  [[fc747bb](https://github.com/h5bp/server-configs-apache/commit/fc747bbdf0a0c224ec08d8b925f33671e4d5046d)].
+* Drop Bower support
+  [[ee6cd75](https://github.com/h5bp/server-configs-apache/commit/ee6cd751f0b907239a032ec5477ee3bfbc2bc570)].
+* Fix HTTPS enforcement rule
+  [[11e523d](https://github.com/h5bp/server-configs-apache/commit/11e523d10ad8bb604fe692ec8d1fd40adc0010fa)].
+* Improve inline comments.
```
Comment (mathiasbynens) [inline CHANGELOG.md]: No link for this item?

**Comment by mathiasbynens** [comment]
LGTM modulo nits 👍 Thanks for doing this, @LeoColomb!

File: CHANGELOG.md (hunk 1 above)
Comment (LeoColomb) [reply CHANGELOG.md]: Well, no, there are [a lot of small fixes](https://github.com/h5bp/server-configs-apache/compare/2.14.0...master) then I did the [same as before](https://github.com/h5bp/server-configs-apache/blob/8ab23cd794798d41bb660751df1f8ab90445e679/CHANGELOG.md#2100-october-20-2014). 😉

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "author": "The H5BP Team",
   "description": "Boilerplate configurations for the Apache HTTP server",
-  "version": "2.14.0",
+  "version": "2.15.0",
```
Comment (XhmikosR) [inline package.json]: Can you change the version in packge-lock.json too?

Comment (LeoColomb) [reply package.json]: Oh, good catch!

Comment (LeoColomb) [reply package.json]: Done 👍

**Comment by LeoColomb** [comment]
Then we need an admin to publish it on npm. Maybe @mathiasbynens? Thx!

**Comment by LeoColomb** [comment]
@alrra? You seem to be the [only collaborator on npm](https://www.npmjs.com/package/apache-server-configs/access) 🤞 
Thx!

**Pull request merged**